GitUI/CommandsDialogs/SettingsDialog/Pages/ConsoleStyleSettingsPage.Designer.cs

```diff
@@ -0,0 +1,158 @@
+            this.groupBoxConsoleSettings.Padding = new System.Windows.Forms.Padding(3, 2, 3, 2);
+            this.groupBoxConsoleSettings.Size = new System.Drawing.Size(1706, 89);
+            this.groupBoxConsoleSettings.TabIndex = 4;
+            this.groupBoxConsoleSettings.TabStop = false;
+            this.groupBoxConsoleSettings.Text = "Console settings (restart required)";
+            // 
+            // label3
+            // 
+            this.label3.AutoSize = true;
+            this.label3.Location = new System.Drawing.Point(6, 54);
+            this.label3.Name = "label3";
+            this.label3.Size = new System.Drawing.Size(49, 13);
+            this.label3.TabIndex = 7;
+            this.label3.Text = "Font size";
+            // 
+            // cboFontSize
+            // 
+            this.cboFontSize.FormattingEnabled = true;
+            this.cboFontSize.Items.AddRange(new object[] {
+            "8",
+            "9",
+            "10",
+            "11",
+            "12",
+            "13",
+            "14",
+            "16",
+            "18",
+            "19",
+            "20",
+            "24"});
+            this.cboFontSize.Location = new System.Drawing.Point(118, 51);
+            this.cboFontSize.Margin = new System.Windows.Forms.Padding(3, 2, 3, 2);
+            this.cboFontSize.Name = "cboFontSize";
+            this.cboFontSize.Size = new System.Drawing.Size(262, 21);
+            this.cboFontSize.TabIndex = 6;
+            // 
+            // label1
+            // 
+            this.label1.AutoSize = true;
+            this.label1.Location = new System.Drawing.Point(5, 28);
+            this.label1.Name = "label1";
+            this.label1.Size = new System.Drawing.Size(69, 13);
+            this.label1.TabIndex = 5;
+            this.label1.Text = "Console style";
+            // 
+            // _NO_TRANSLATE_cboStyle
+            // 
+            this._NO_TRANSLATE_cboStyle.DropDownStyle = System.Windows.Forms.ComboBoxStyle.DropDownList;
+            this._NO_TRANSLATE_cboStyle.FormattingEnabled = true;
+            this._NO_TRANSLATE_cboStyle.Items.AddRange(new object[] {
+            "Default",
+            "<Default Windows scheme>",
+            "<Base16>",
+            "<Cobalt2>",
+            "<ConEmu>",
+            "<Gamma 1>",
+            "<Monokai>",
+            "<Murena scheme>",
+            "<PowerShell>",
+            "<Solarized>",
+            "<Solarized Git>",
+            "<Solarized (Luke Maciak)>",
+            "<Solarized (John Doe)>",
+            "<Solarized Light>",
+            "<SolarMe>",
+            "<Standard VGA>",
+            "<tc-maxx>",
+            "<Terminal.app>",
+            "<Tomorrow>",
+            "<Tomorrow Night>",
+            "<Tomorrow Night Blue>",
+            "<Tomorrow Night Bright>",
+            "<Tomorrow Night Eighties>",
+            "<Twilight>",
+            "<Ubuntu>",
+            "<xterm>",
+            "<Zenburn>"});
```

Comment:
Console styles

Reply:
Had a quick look but didn't find a way to do it.
It seems that ConEmu don't expose this kind of data.
And even if it do it, I don't think it should be done in this PR because we would have to change a lot of source code to expose it in the settings page.